api-reference/lease/find-leases-for-tenant.mdx

```diff
@@ -163,7 +182,7 @@ Returned if none of the tenant parameters are provided in the request body
 {
    "error": {
     "type": "no_tenant_information",
-    "message": "You need to provide at least one of identifier, email_address or phone_number in the request body."
+    "message": "You need to provide at least one of identifier, external_ref AND group, email_address or phone_number in the request body."
```

Comment:
is that "AND" meant to be capitalized?

Reply:
Suggest an Oxford comma for this one (comma in bold): "You need to provide at least one of identifier, external_ref AND group, email_address**,** or phone_number in the request body."

Reply:
Yes, I wanted to emphasise that you must provide an external ref and a group if you choose to identify a tenant via external ref. I've added a comma to increase clarity.